Sources/EduIDExpansion/Flows/PersonalInfo/PersonalInfoViewController.swift

```diff
@@ -0,0 +1,94 @@
+import UIKit
+import TinyConstraints
+
+class PersonalInfoViewController: UIViewController, ScreenWithScreenType {
+   
+    //MARK: screen type
+    var screenType: ScreenType = .personalInfoLandingScreen
+    
+    //MARK: - delegate
+    weak var delegate: PersonalInfoViewControllerDelegate?
+    
+    private let scrollView = UIScrollView()
```

Comment:
Why is this scroll view stored in a property? It seems unused outside of the setupUI method, and it would stay alive as child of the view.

Reply:
true I removed the property